index.js

```diff
@@ -0,0 +1,145 @@
+'use strict';
+
+const NR = require('node-resque');
+const asCallback = require('ascallback');
+const config = require('config');
+const winston = require('winston');
+const request = require('request');
+const ecosystem = config.get('ecosystem');
+const executorConfig = config.get('executor');
+const redisConfig = config.get('redis');
+const ExecutorPlugin = require('screwdriver-executor-router');
+const executorPlugins = Object.keys(executorConfig).reduce((aggregator, keyName) => {
```

Comment:
It's confusing to have two variables named almost the same apart from case. Can we rename one of them please?